.dockerignore

```diff
@@ -6,7 +6,6 @@
 !geoportal
 !admin
 !scripts/clone_schema.sql
-admin/Makefile
```

Comment:
Why this? For me we should keep that... and add:
```
admin/.env
admin/Dockerfile
admin/Makefile
admin/README.md
admin/config.yaml
admin/docker-compose.yaml
```
Or inverse the filter (probable should be preferred)
Remove `!admin`
and add
```
!admin/package.json
!admin/c2cgeoportal_admin
!admin/requirements.txt
!admin/setup.cfg
!admin/setup.py
```

Reply:
Good catch, I got lost with such contradictions (not all but admin but not ...) the second option seems clearer.